packages/components/src/components/assets/CoinLogo/CoinLogo.tsx

```diff
@@ -24,15 +25,44 @@ const SvgWrapper = styled.div<Omit<CoinLogoProps, 'symbol'>>`
     }
 `;
 
-export const CoinLogo = ({ symbol, className, size = 32, ...rest }: CoinLogoProps) => (
-    <SvgWrapper className={className} size={size} {...rest}>
-        <ReactSVG
-            src={COINS[symbol]}
-            beforeInjection={svg => {
-                svg.setAttribute('width', `${size}px`);
-                svg.setAttribute('height', `${size}px`);
-            }}
-            loading={() => <span className="loading" />}
-        />
-    </SvgWrapper>
+const StyledIconImage = styled.img<CoinLogoProps>(
```

Comment:
Same situation here with the transient Props

Reply:
Event if this wont generate error the absence of `$` prefix will cause the `size` prop to be leaked into HTML DOM.